packages/salesforcedx-vscode-apex/src/views/testOutlineProvider.ts

```diff
@@ -40,9 +43,9 @@ export class ApexTestOutlineProvider
   private path: string;
   private apexTestInfo: ApexTestMethod[] | null;
 
-  constructor(path: string, apexTestInfo: ApexTestMethod[] | null) {
+  constructor(xpath: string, apexTestInfo: ApexTestMethod[] | null) {
```

Comment:
`xpath` aka `this.path` seems to be exclusively used for `this.getAllApexTests` which really doesn't need that parameter. Please check if `this.path` is used somewhere other than `getAllApexTests` and remove it if it's not needed.

Reply:
Okay I'll check for that